tests/ui/feature-gates/feature-gate-concat_idents2.stderr

```diff
@@ -7,11 +7,11 @@ LL |     concat_idents!(a, b);
    = note: see issue #29599 <https://github.com/rust-lang/rust/issues/29599> for more information
    = help: add `#![feature(concat_idents)]` to the crate attributes to enable
 
-error[E0425]: cannot find value `ab` in this scope
+error[E0425]: cannot find value `ab` in the expanded code of procedural macro `concat_idents`
   --> $DIR/feature-gate-concat_idents2.rs:2:5
    |
 LL |     concat_idents!(a, b);
-   |     ^^^^^^^^^^^^^^^^^^^^ not found in this scope
+   |     ^^^^^^^^^^^^^^^^^^^^ not found in the expanded code of procedural macro `concat_idents`
```

Comment:
this could use a less redundant message. Maybe something about `from this macro invocation`

Reply:
changed